src/cmake/dependencies.cmake

```diff
@@ -112,23 +117,28 @@ if(NOT USE_SYSTEM_MAGNUM)
   set(WITH_STBIMAGEIMPORTER ON CACHE BOOL "WITH_STBIMAGEIMPORTER" FORCE)
   set(WITH_STBIMAGECONVERTER ON CACHE BOOL "WITH_STBIMAGECONVERTER" FORCE)
   set(WITH_SDL2APPLICATION OFF CACHE BOOL "WITH_SDL2APPLICATION" FORCE)
+  set(WITH_GLFWAPPLICATION OFF CACHE BOOL "WITH_GLFWAPPLICATION" FORCE)
   set(WITH_EIGEN ON CACHE BOOL "WITH_EIGEN" FORCE) # Eigen integration
-  set(WITH_PYTHON ON CACHE BOOL "" FORCE) # Python bindings
+  if(BUILD_PYTHON_BINDINGS)
+    set(WITH_PYTHON ON CACHE BOOL "" FORCE) # Python bindings
```

Comment:
Hmm maybe also doing 

```cmake
if(BUILD_PYTHON_BINDINGS)
  add_subdirectory("${DEPS_DIR}/magnum-bindings")
endif()
```

down below, instead of adding it unconditionally?